app/services/StripeService.scala

```diff
@@ -100,6 +100,8 @@ class StripeService(
     idUser.foreach { id =>
       identityService.updateMarketingPreferences(id, marketing)
     }
+    if (marketing)
```

Comment:
we can remove this, plus lines 100 to 102, plus `marketing` param from this function